### PR TITLE
Update Refresh from Flapjack Branch

### PR DESCRIPTION
**Pull request by kave**
### Added
Add global context variable `global_dict` for easier prototyping
Add styleguide app to local settings
Add S3 upload capability

### CODE REVIEW

File: cfgov/cfgov/settings/base.py [hunk 1]
```diff
@@ -259,3 +260,14 @@
 
 # PDFReactor
 PDFREACTOR_LIB = os.environ.get('PDFREACTOR_LIB', '/opt/PDFreactor/wrappers/python/lib')
+
+# S3 Configuration
+if os.environ.get('S3_ENABLED', False):
+    DEFAULT_FILE_STORAGE = 'v1.s3utils.MediaRootS3BotoStorage'
+    AWS_S3_SECURE_URLS = False  # True = use https; False = use http
+    AWS_QUERYSTRING_AUTH = False  # don't add complex authentication-related query parameters for requests
```
Comment (anselmbradford) [inline cfgov/cfgov/settings/base.py]: Very minor, but this code comment could be updated to follow the format in L:267 for clarity. If at some point someone is in here.

File: cfgov/jinja2/v1/_includes/molecules/related-posts.html [hunk 2]
```diff
@@ -34,6 +34,8 @@ <h2 class="header-slug">
         </span>
     </h2>
     {% endif %}
+    {% if global_dict %} print sometihng {% endif %}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/related-posts.html]: `print sometihng` ?? What's this?

Comment (kave) [inline cfgov/jinja2/v1/_includes/molecules/related-posts.html]: lol strange, good catch i'll remove this in a quick fix

File: cfgov/cfgov/settings/base.py (hunk 1 above)
Comment (kave) [inline cfgov/cfgov/settings/base.py]: sure no problem

**Pull request merged**